lib/exec.js

```diff
@@ -59,11 +61,17 @@ function exec (cmd, args = [], opts = {}) {
       if (proc.stdout) {
         proc.stdout.on('data', (data) => {
           stdoutArr.push(data);
+          if (opts.logger && _.isFunction(opts.logger.debug)) {
+            opts.logger.debug(data);
+          }
         });
       }
       if (proc.stderr) {
         proc.stderr.on('data', (data) => {
           stderrArr.push(data);
+          if (opts.logger && _.isFunction(opts.logger.error)) {
+            opts.logger.error(data);
```

Comment:
some processes put "normal" data into stderr (like ffmpeg does). I'd rather decrease the logging level here

Reply:
Wouldn't that cause real errors to get mis-logged? 

I would rather not make this badly handle the normal case just for some badly made subprograms. Instead, the logger sent in, in such a case, should link the `error` logger to the one that is wanted:
```js
let ffmpegLogger = logger.getLogger('ffmpeg');
ffmpegLogger.error = ffmpegLogger.debug; // or whatever level
await exec('ffmpeg', [], {logger: ffmpegLogger});
```